CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### BREAKING CHANGES
```

Comment:
This means that this PR will enter v7

Reply:
Yes, right because it will change the interface making it breaking.